README.md

```diff
@@ -39,7 +39,7 @@ If you're not calling `cargo` directly and cannot change how it's invoked, you c
 
 Microsoft uses `cargo auditable` internally and maintains the [data extraction library for Go](https://github.com/microsoft/go-rustaudit).
 
```

Comment:
```suggestion

Chainguard includes `cargo auditable` in their rust base container, with a default `cargo` wrapper to always call `cargo auditable`, so that Rust applications built using this container are auditable by default.
```

Reply:
That's nice, and maybe make chainguard a url to either https://chainguard.dev or to https://images.chainguard.dev/directory/image/rust/overview

Reply:
Committed in https://github.com/rust-secure-code/cargo-auditable/pull/220/commits/6cbc395272552dcc4eec5c621c529c4769e9cfae :)